ui/app/pages/confirm-decrypt-message/confirm-decrypt-message.component.js

```diff
@@ -0,0 +1,330 @@
+      hasDecrypted,
+      rawMessage,
+      copyToClipboardPressed,
+    } = this.state
+
+    return (
+      <div className="request-decrypt-message__body">
+        { this.renderAccountInfo() }
+        <div
+          className="request-decrypt-message__visual"
+        >
+          <section>
+            {origin.icon ? (
+              <img
+                className="request-decrypt-message__visual-identicon"
+                src={origin.icon}
+              />
+            ) : (
+              <i className="request-decrypt-message__visual-identicon--default">
+                {origin.name.charAt(0).toUpperCase()}
+              </i>
+            )}
+            <div
+              className="request-decrypt-message__notice"
+            >
+              { notice }
+            </div>
+          </section>
+        </div>
+        <div
+          className="request-decrypt-message__message"
+        >
+          <div
+            className="request-decrypt-message__message-text"
+          >
+            { !hasDecrypted ? txData.msgParams.data : rawMessage }
+          </div>
+          <div
+            className={classnames({
+              'request-decrypt-message__message-cover': true,
+              'request-decrypt-message__message-lock--pressed': hasDecrypted,
+            })}
+          >
+          </div>
+          <div
+            className={classnames({
+              'request-decrypt-message__message-lock': true,
+              'request-decrypt-message__message-lock--pressed': hasDecrypted,
+            })}
+            onClick={(event) => {
+              this.props.decryptMessageInline(txData, event).then((result, err) => {
+                if (!err) {
+                  this.setState({ hasDecrypted: true, rawMessage: result.rawData })
+                } else {
+                  this.setState({ hasDecrypted: true, rawMessage: err })
+                }
+              })
+            }}
+          >
+            <img src="images/lock.svg" />
+            <div
+              className="request-decrypt-message__message-lock-text"
+            >
+              {this.context.t('decryptMetamask')}
+            </div>
+          </div>
+        </div>
+        { hasDecrypted ?
+          (
+            <div
+              className={classnames({
+                'request-decrypt-message__message-copy': true,
+                'request-decrypt-message__message-copy--pressed': copyToClipboardPressed,
+              })}
+              onClick={() => this.copyMessage()}
+              onMouseDown={() => this.setState({ copyToClipboardPressed: true })}
+              onMouseUp={() => this.setState({ copyToClipboardPressed: false })}
+            >
+              <Tooltip
+                position="bottom"
+                title={hasCopied ? this.context.t('copiedExclamation') : this.context.t('copyToClipboard')}
+                wrapperClassName="request-decrypt-message__message-copy-tooltip"
+              >
+                <div
```

Comment:
This "copy to clipboard" part should be centred horizontally, and should be this color: `#037DD6`